### PR TITLE
[T3A3][W09-A4]Muhammad Mustaqiim Bin Muhar

### PR DESCRIPTION
**Pull request by mustaqiimuhar**

### CODE REVIEW

**Comment by mustaqiimuhar** [comment]
Ready for review.

File: doc/UserGuide.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > Persons can have any number of tags (including 0)
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
+* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01, 123456`
```
Comment (hwkchia) [inline doc/UserGuide.md:48]: 👍 Great! You remembered to change the user guide...

File: src/seedu/addressbook/commands/AddCommand.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
             + "Contact details can be marked private by prepending 'p' to the prefix.\n"
             + "Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...\n"
             + "Example: " + COMMAND_WORD
-            + " John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney";
```
Comment (hwkchia) [inline src/seedu/addressbook/commands/AddCommand.java:22]: ... as well as the program help.

File: src/seedu/addressbook/data/person/Address.java [hunk 3]
```diff
@@ -3,57 +3,99 @@
...
+	}
+
+	@Override
+	public int hashCode() {
+		return toString().hashCode();
+	}
+
+	public boolean isPrivate() {
+		return isPrivate;
+	}
```
Comment (hwkchia) [inline src/seedu/addressbook/data/person/Address.java:100]: I think the indentation is causing some issues here.  Seems entire chunk of code is changed.

**Review by hwkchia — COMMENTED** [review]
Good effort in creating classes.  But do remember to comment your classes also.  Code quality should always be at the back of your mind.

**Pull request closed without merging**